convert/convert.go

```diff
@@ -114,11 +115,27 @@ func stringifyExpression(tokens hclwrite.Tokens) string {
 }
 
 func AppendLocalsBlock(file *hclwrite.File, filename string, terratag TerratagLocal) {
+	key := tag_keys.GetTerratagAddedKey(filename)
+
+	// If there's an existings terratag locals, remove it.
+	// A merged one will be added instead.
+	blocks := file.Body().Blocks()
```

Comment:
See the comment. But basically before appending a new merged terratag locals, remove the old one (if exist).

Reply:
Something I might be missing here.
This code basically creates a terratag `locals` block, and remove the old terratag `locals` block beforehand if it already exists.

So:

1. This isn't really an `AppendLocalsBlock`, right? More of a "reset" locals block
2. There's more code here that merges non-terratag `locals` block with terratag `locals` block. Is that necessary?

Reply:
@razbensimon - you're correct about 1. I think some names changes are due.
About 2: No, the code doesn't merge non-terratag locals (it's not necessary and probably a bad idea). It only merges the terratag locals. This is to fix issues such as a corrupted HCL if running terratag more than once. (again probably naming is confusing and will be fixed).

Reply:
We've talked, I've misunderstood, thought the purpose here was to specifically overwrite the terratag local, and not merge it with previous values. That's a good solution as well. Let's go with that 👍🏻